pytket/extensions/quantinuum/backends/leakage_gadget.py

```diff
@@ -174,5 +174,5 @@ def prune_shots_detected_as_leaky(result: BackendResult) -> BackendResult:
                 for key, val in discarded_counts.items()
             }
         ),
-        c_bits=cast(Sequence[Bit], cast(Sequence[Bit], regular_bits)),
+        c_bits=cast(Sequence[Bit], regular_bits),  # type: ignore
```

Comment:
surprised the `type: ignore` is still needed with the cast?

Reply:
I'll double check! I think it won't be, I added the type ignore before removing the extra cast.

Reply:
yeah it's fine with the type ignore removed now.